packages/synthetix-governance/tasks/fast-forward.js

```diff
@@ -0,0 +1,32 @@
+const { task } = require('hardhat/config');
+const types = require('@synthetixio/core-js/utils/hardhat/argument-types');
+const { fastForward } = require('@synthetixio/core-js/utils/hardhat/rpc');
+
+const units = {
+  seconds: 1,
+  minutes: 60,
+  hours: 60 * 60,
+  days: 60 * 60 * 24,
+  weeks: 60 * 60 * 24 * 7,
+};
+
+task('fast-forward', 'travel the given amount of time on the current node')
+  .addPositionalParam(
+    'amount',
+    'The amount of time that you want to travel, by default in seconds.',
+    undefined,
+    types.int
+  )
+  .addOptionalPositionalParam(
+    'unit',
+    `Unit of time, should be one of (${Object.keys(units).join('|')})`,
+    'seconds',
```

Comment:
Should we default to days? This will be more common for ElectionModule instance defaults.

Reply:
Yup, makes sense 🚀